### PR TITLE
Migration steps from Azure Polling to Azure Monitor

### PR DESCRIPTION
**Pull request by mcombina-nr**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
We are adding steps tog guide customers using Azure Polling to switch to Azure Monitor
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx [hunk 1]
```diff
@@ -5,12 +5,15 @@ redirects:
 ---
 
 import infrastructureAzureMonitorSelect from 'images/azure-monitor.webp'
+import infrastructureAzureMonitorMetricsIntegration from 'images/azure-monitor-metrics.webp'
+import infrastructureAzureMonitorMetricsConfigure from 'images/azure-monitor-configure.webp'
+import infrastructureAzureMonitorEnableToggle from 'images/azure-monitor-enable-toggle.webp'
```
Comment (mcombina-nr) [inline src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx:11]: Adding 3 new screenshots

File: src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx [hunk 2]
```diff
@@ -5,12 +5,15 @@ redirects:
...
 
 Monitor and report data about your Microsoft Azure services to New Relic with the New Relic Azure Monitor integration.
 
 ## Features [#features]
 
-Our Azure Monitor integration monitors all your Azure metrics from all supported Azure services. Once you enable it, we query your Azure platform services according to the polling interval you set. Once enabled, you get: 
+Our Azure Monitor integration monitors all your Azure metrics from all supported Azure services. Once you enable it, we query your Azure platform services according to the polling interval you set. Once enabled, you get:
```
Comment (mcombina-nr) [inline src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx:17]: A lot of these changes are just removing the trailing spaces.

File: src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx [hunk 3]
```diff
@@ -265,3 +268,39 @@ When migrating from our traditional Azure integrations to the Azure Monitor inte
...
+/>
+
+3. The "Azure Monitor metrics" integration is now available
+<img
+  title="Shows where azure monitor metrics integration"
+  alt="Shows where azure monitor metrics integration"
+  src={infrastructureAzureMonitorMetricsConfigure}
+/>
+
+4. Enable and configure the new integration as described in [this section](/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor/#set-up)
```
Comment (mcombina-nr) [inline src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx:291]: This is a link to a different section of the same page, there must be a better way to do it

File: src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx [hunk 4]
```diff
@@ -5,12 +5,15 @@ redirects:
 ---
 
 import infrastructureAzureMonitorSelect from 'images/azure-monitor.webp'
+import infrastructureAzureMonitorMetricsIntegration from 'images/azure-monitor-metrics.webp'
+import infrastructureAzureMonitorMetricsConfigure from 'images/azure-monitor-configure.webp'
+import infrastructureAzureMonitorEnableToggle from 'images/azure-monitor-enable-toggle.webp'
```
Comment (mcombina-nr) [inline src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx:10]: Adding 3 new screenshots

File: src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx [hunk 5]
```diff
@@ -5,12 +5,15 @@ redirects:
...
 
 Monitor and report data about your Microsoft Azure services to New Relic with the New Relic Azure Monitor integration.
 
 ## Features [#features]
 
-Our Azure Monitor integration monitors all your Azure metrics from all supported Azure services. Once you enable it, we query your Azure platform services according to the polling interval you set. Once enabled, you get: 
+Our Azure Monitor integration monitors all your Azure metrics from all supported Azure services. Once you enable it, we query your Azure platform services according to the polling interval you set. Once enabled, you get:
```
Comment (mcombina-nr) [inline src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx:16]: Some diffs are only removing the trailing space.

File: src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx [hunk 6]
```diff
@@ -265,3 +268,39 @@ When migrating from our traditional Azure integrations to the Azure Monitor inte
 
 * When you enable an Azure Monitor integration, new separate entities will be created for all of your resources. The entities created by the Azure Polling integration are staying as they are. This means you must update dashboards, alerts, and any other capability that refrences those entities.
 * Old entities are available for 24 hours.
+
```
Comment (mcombina-nr) [inline src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx:271]: Starting here, and to the end of the PR: these are the new steps added.
Please review carefully.

**Review by jeff-colucci — DISMISSED** [review]
Everything that I've seen looks good after some minor formatting and style changes. Once the build finishes, I'll do a final check and publish

File: src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx [hunk 7]
```diff
@@ -9,11 +9,11 @@ import infrastructureAzureMonitorMetricsIntegration from 'images/azure-monitor-m
 import infrastructureAzureMonitorMetricsConfigure from 'images/azure-monitor-configure.webp'
 import infrastructureAzureMonitorEnableToggle from 'images/azure-monitor-enable-toggle.webp'
 
-Monitor and report data about your Microsoft Azure services to New Relic with the New Relic Azure Monitor integration.
+Monitor and report data about your Microsoft Azure services to New Relic with the New Relic Azure monitor integration.
```
Comment (mcombina-nr) [inline src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx:12]: Is 'monitor' the right casing? MSFT uses 'Monitor'(https://learn.microsoft.com/en-us/azure/azure-monitor/overview, https://docs.newrelic.com/docs/release-notes/infrastructure-release-notes/cloud-integration-release-notes/) and New Relic also uses 'Monitor' (Infrastructure -> Azure. The integration is called 'Azure Monitor metrics').
NR release notes also uses 'Monitor': https://docs.newrelic.com/docs/release-notes/infrastructure-release-notes/cloud-integration-release-notes/

File: src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx [hunk 8]
```diff
@@ -264,43 +264,42 @@ The following table shows the differences between both solutions:
...
 
 <Callout variant="important">
-  If both integrations are enabled for the same resource type, you may see duplicate metrics.
-  This can happen if you use a generic name field to filter your results that can be shared from the Azure Monitor and the other polling Azure polling integrations.
-  The results from multiple resources could be joined unintentionally.
-  Note that metrics can still be told apart by faceting using the field `collector.name`.
-  Rate limits as enforced by Azure will be shared between any current polling integrations and Azure Monitor.
+  If both integrations are enabled for the same resource type, you may see duplicate metrics. This can happen if you use a generic name field to filter your results that can be shared from the Azure monitor and the other polling Azure polling integrations. The results from multiple resources could be joined unintentionally.
+  
+Note that metrics can still be differentiated by faceting using the field `collector.name`. Also note that rate limits as enforced by Azure will be shared between any current polling integrations and the Azure monitor.
```
Comment (mcombina-nr) [inline src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx:304]: Much better, thanks!

**Review by jeff-colucci — DISMISSED** [review]

**Comment by jeff-colucci** [comment]
Build successful at [this Gatsby page](https://www.gatsbyjs.com/dashboard/34e708d2-68a1-48ca-b66f-d2604890b061/sites/a0b77314-cdea-4790-8fde-7d3457d590ce/builds/6bb2cc5e-4fa7-49a6-a85f-532517683c1b/details)

**Pull request merged**